api/pkg/template/ca.go

```diff
@@ -0,0 +1,10 @@
+package template
+
+import "github.com/replicatedhq/embedded-cluster/pkg/addons/adminconsole"
+
+// privateCACert returns the name of a ConfigMap containing additional CA certificates
+// provided by the host system. In Embedded Cluster, this ConfigMap is automatically
+// created and managed when a host CA bundle is detected.
+func (e *Engine) privateCACert() string {
+	return adminconsole.PrivateCASConfigMapName
```

Comment:
I believe this should return an empty string for target==kubernetes installations until we support installing with your own private ca